src/styles/challenges.scss

```diff
@@ -0,0 +1,76 @@
+.carouselItem{
```

Comment:
This is a small thing but - in all of the other sections, we have been naming the CSS classes using [BEM syntax](https://csswizardry.com/2013/01/mindbemding-getting-your-head-round-bem-syntax/) - so classes are named like `carousel__item` or `carousel__caption` with the double underscore after the section description. Would you mind switching to that for consistency?